app/helpers/users_helper.rb

```diff
@@ -1,2 +1,5 @@
 module UsersHelper
+  def taunt1
+    pp "taunt"
```

Comment:
```suggestion
    pp 'taunt'
```



<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>